### PR TITLE
modify some comments for consistency

### PR DESCRIPTION
**Pull request by windsonsea**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

- Fix typos
- Keep comments consistent in a file

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/kwok/cni/netns_linux.go [hunk 1]
```diff
@@ -135,18 +135,18 @@ func NewNS(name string) (ns.NetNS, error) {
...
 		}
 
-		// Put this thread back to the orig ns, since it might get reused (pre go1.10)
+		// Put this thread back to the origNS, since it might get reused (pre go1.10).
 		defer func() {
 			_ = origNS.Set()
 		}()
 
-		// bind mount the netns from the current thread (from /proc) onto the
+		// Bind-mount the netns from the current thread (from /proc) onto the
```
Comment (wzshiming) [inline pkg/kwok/cni/netns_linux.go:149]:
```suggestion
		// Bind mount the netns from the current thread (from /proc) onto the
```

File: pkg/kwok/cni/netns_linux.go [hunk 2]
```diff
@@ -61,17 +61,17 @@ func GetNS(name string) (ns.NetNS, error) {
...
+// NewNS creates a new persistent (bind-mounted) network namespace and
+// returns an object representing that namespace, without switching to it.
 func NewNS(name string) (ns.NetNS, error) {
 	nsRunDir := getNsRunDir()
 
 	nsName := fmt.Sprintf("cni-kwok-%s", name)
 	nsPath := path.Join(nsRunDir, nsName)
 
-	// Create the directory for mounting network namespaces
+	// Create the directory for mounting network namespaces.
```
[the remark this answers is not in the document]
Comment (wzshiming) [reply pkg/kwok/cni/netns_linux.go:72]: For the user-visible part, I think it is still more important.

File: pkg/kwok/cni/netns_linux.go [hunk 3]
```diff
@@ -87,44 +87,44 @@ func NewNS(name string) (ns.NetNS, error) {
...
 	if err != nil {
 		return nil, err
 	}
 	_ = mountPointFd.Close()
 
-	// Ensure the mount point is cleaned up on errors; if the namespace
+	// Ensure the mount point is cleaned up on errors. If the namespace
 	// was successfully mounted this will have no effect because the file
-	// is in-use
+	// is in use.
```
[the remark this answers is not in the document]
Comment (windsonsea) [reply pkg/kwok/cni/netns_linux.go:112]: `be in use` is more common really. Whatever, that depends on what your like 😄

**Review by wzshiming — APPROVED** [review]
/approve
/lgtm

**Pull request merged**